src/library-authoring/components/ContainerDeleter.test.tsx

```diff
@@ -0,0 +1,205 @@
+        expect(mockDelete).toHaveBeenCalled();
+      });
+      expect(mockCancel).toHaveBeenCalled(); // In order to close the modal, this also gets called.
+      expect(mockShowToast).toHaveBeenCalled();
+      // Get restore / undo func from the toast
+      const restoreFn = mockShowToast.mock.calls[0][1].onClick;
+      restoreFn();
+      await waitFor(() => {
+        expect(mockRestore).toHaveBeenCalled();
+        expect(mockShowToast).toHaveBeenCalledWith('Undo successful');
+      });
+    });
+
+    it(`<${context}> should show subsection message if parent data is set with one parent`, async () => {
+      const mockCancel = jest.fn();
+      const { containerId, parent } = getContainerDetails(context);
+      if (!parent) {
+        return;
+      }
+      mockSearchResult({
+        results: [ // @ts-ignore
+          {
+            hits: [{
+              [`${parent}s`]: {
+                displayName: [`${parent} 1`],
+                key: [`${parent}1`],
+              },
+              blockType: context,
+            }],
+          },
+        ],
+      });
+      render(
+        <ContainerDeleter
+          containerId={containerId}
+          displayName={`Test ${context}`}
+          isOpen
+          close={mockCancel}
+        />,
+        renderArgs,
+      );
+
+      const modal = await screen.findByRole('dialog', { name: new RegExp(`Delete ${context}`, 'i') });
+      expect(modal).toBeVisible();
+
+      const textMatch = new RegExp(`By deleting this ${context}, you will also be deleting it from ${parent} 1 in this library.`);
+      expect((await screen.findAllByText((_, element) => textMatch.test(element?.textContent || ''))).length).toBeGreaterThan(0);
+    });
+
+    it(`<${context}> should show parents message if parents is set with multiple parents`, async () => {
+      const mockCancel = jest.fn();
+      const { containerId, parent } = getContainerDetails(context);
+      if (!parent) {
+        return;
+      }
+      mockSearchResult({
+        results: [ // @ts-ignore
+          {
+            hits: [{
+              [`${parent}s`]: {
+                displayName: [`${parent} 1`, `${parent} 2`],
+                key: [`${parent}1`, `${parent}2`],
+              },
+              blockType: context,
+            }],
+          },
+        ],
+      });
+      render(
+        <ContainerDeleter
+          containerId={containerId}
+          displayName={`Test ${context}`}
+          isOpen
+          close={mockCancel}
+        />,
+        renderArgs,
+      );
+
+      const modal = await screen.findByRole('dialog', { name: new RegExp(`Delete ${context}`, 'i') });
+      expect(modal).toBeVisible();
+
+      const textMatch = new RegExp(`By deleting this ${context}, you will also be deleting it from 2 ${parent}s in this library.`, 'i');
+      expect((await screen.findAllByText((_, element) => textMatch.test(element?.textContent || ''))).length).toBeGreaterThan(0);
+    });
+  });
+});
```

Comment:
I think we can also reinstate this [skipped test](https://github.com/openedx/frontend-app-authoring/blob/4b4ab923837a77566c934cc93a39278961349697/src/library-authoring/containers/ContainerInfo.test.tsx#L69).